.github/CONTRIBUTING.md

```
@@ -0,0 +1,34 @@
+$ composer pcov     // pcov
+```
+
+Add tests for your new code ensuring that you have 100% code coverage.
+In rare cases, code may be excluded from test coverage using `@codeCoverageIgnore`.
+
+## Sending a pull request
+
+To ensure your PHP code changes pass the CI checks, make sure to run all the same checks before submitting a PR.
+
+```bash
+$ composer tests
```

Markdownlint (0.37.0)

Dollar signs used before commands without showing output

(MD014, commands-show-output)